modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -526,8 +520,14 @@ private Comparator<RasRunResult> buildRunsComparator(RasQueryParameters queryPar
         return runsComparator;
     }
 
-	private boolean isTestRunShort(boolean isMethodDetailsRequested) {
-		return isMethodDetailsRequested ? false : true;
+	private boolean isTestRunShort(String detailParam) {
+		
+		boolean isShort = true;
+		if(detailParam != null && detailParam.equals(DETAILS_METHOD_QUERY_PARAM_VALUE)) {
+			isShort = false;
+		}
+
+		return isShort;
```

Comment:
I think this won't be needed if we add a method inside the RasDetailsQueryParams class that returns true if we've passed in "methods" into the details param.